<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Metrochat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>841</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Start</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Tab1">
     <attribute name="title">
      <string>Chat</string>
     </attribute>
     <widget class="QPlainTextEdit" name="TextInput">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>300</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="ChatOutput">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>451</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SendButton">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>330</y>
        <width>85</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLabel" name="StatusTittle">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>66</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="StatusLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>380</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="IPInput">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="plainText">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="IPTittle">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>21</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="PortInput">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="plainText">
       <string>25000</string>
      </property>
     </widget>
     <widget class="QLabel" name="PortTittle">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>31</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ConnectionButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QLabel" name="ServerClientLabel">
      <property name="geometry">
       <rect>
        <x>81</x>
        <y>230</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Working as server</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ServerButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>180</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Server</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="ClientButton">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>180</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Client</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="ConfigurationTittle">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Configuration</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="UserInput">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="plainText">
       <string>Andres</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserTittle">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>41</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>User</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Tab2">
     <attribute name="title">
      <string>Send files</string>
     </attribute>
     <widget class="QLabel" name="ConfigurationTittle_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Configuration</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="PortInput_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="plainText">
       <string>25000</string>
      </property>
     </widget>
     <widget class="QLabel" name="IPTittle_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>21</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="ServerClientLabel_2">
      <property name="geometry">
       <rect>
        <x>81</x>
        <y>230</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Working as server</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="UserInput_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="plainText">
       <string>Andres</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="IPInput_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="plainText">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ClientButton_2">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>180</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Client</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="ConnectionButton_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ServerButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>180</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Server</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="UserTittle_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>41</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>User</string>
      </property>
     </widget>
     <widget class="QLabel" name="PortTittle_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>31</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="StatusTittle_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="StatusLabel_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>380</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="RouteButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>60</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Route</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="RouteOutput">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SendFileButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>110</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLabel" name="RouteLabel">
      <property name="geometry">
       <rect>
        <x>261</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>File path:</string>
      </property>
     </widget>
     <widget class="QLabel" name="ProcessBarLabel">
      <property name="geometry">
       <rect>
        <x>266</x>
        <y>110</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Process:</string>
      </property>
     </widget>
     <widget class="QLabel" name="StatusLabel_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>140</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="StatusTittle_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="RouteOutput_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>330</y>
        <width>341</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="RouteLabel_2">
      <property name="geometry">
       <rect>
        <x>234</x>
        <y>330</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Destiny path:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RouteButton_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>330</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Route</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ResetButton">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>60</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OpenFolderButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>360</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Folder</string>
      </property>
     </widget>
     <widget class="QLabel" name="HelpLabel_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>Path of the file to send.</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="HelpLabel_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>350</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>Destiny folder for the received files.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Push me!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HelpButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
